circuits/precommitChecker.circom

```diff
@@ -0,0 +1,98 @@
+pragma circom 2.1.0;
+
+include "circomlib/poseidon.circom";
+include "circomlib/circuits/comparators.circom";
+include "circomlib/circuits/mimcsponge.circom";
+
+//include "https://github.com/0xPARC/circom-secp256k1/blob/master/circuits/bigint.circom";
+
+//check if m*G = precomp, and hash(m*G) = hash(precomp)
+template singlePrecompCheck() {
+
```

Comment:
make sure not to have empty lines in the beginning of functions — they are quite unnecessary